ocesql/ppout.c

```diff
@@ -1973,13 +1973,13 @@ void ppbuff_incfile(struct cb_exec_list *list){
  		incf = fopen_or_die(filename, "r");
 
 		memset(incmsg, 0, 256);
-		sprintf(incmsg, "%s incfile start:%s", INC_START_MARK, filename);
+		sprintf(incmsg, "%s incfile start:%-s", INC_START_MARK, filename);
```

Comment:
It seems that the update of the line 1976 does not solve warning messages.
https://github.com/yutaro-sakamoto/Open-COBOL-ESQL/runs/5878482122?check_suite_focus=true#step:8:207
```
ppout.c:1976:37: warning: ‘%-s’ directive writing up to 511 bytes into a region of size 169 [-Wformat-overflow=]
 1976 |   sprintf(incmsg, "%s incfile start:%-s", INC_START_MARK, filename);
      |                                     ^~~                   ~~~~~~~~
```
If the warning message seems difficult to solve, please undo the update.